types/node/test/crypto.ts

```diff
@@ -486,6 +486,41 @@ import { promisify } from 'util';
     crypto.createSecretKey(Buffer.from('asdf'));
 }
 
+{
+    const { privateKey, publicKey } = crypto.generateKeyPairSync('ec', {
+        namedCurve: 'sect239k1'
+    });
+
+    const sign: crypto.Signer = crypto.createSign('SHA256');
+    sign.write('some data to sign');
+    sign.end();
+    const signature: string = sign.sign(privateKey, 'hex');
+
+    const verify: crypto.Verify = crypto.createVerify('SHA256');
+    verify.write('some data to sign');
+    verify.end();
+    verify.verify(publicKey, signature);    // $ExpectType boolean
+
+    verify instanceof crypto.Verify;        // $ExpectType boolean
```

Comment:
The type checks here seem a little redundant, TS would error if this didn't work 😛

Reply:
Line 504 is significant. As far as I can tell this is the only line in the test file that will error if `Verify` is changed from a class to a `interface` or `type`.

Reply:
Yep, therefore I added it. The rest is just because there were no tests for these API in place.
But the `// $ExpectType boolean` is redundant and therefore I removed it.